src/Robot.cpp

```diff
@@ -1,26 +1,39 @@
 #include "Robot.h"
 #include "Subsystems/Chassis.h"
 
+#include "Subsystems/BallConveyor.h"
+#include "Subsystems/Intake.h"
+#include "Subsystems/Shooter.h"
+#include "Subsystems/Turret.h"
+#include "Subsystems/Vision.h"
+#include "Subsystems/Lifter.h"
+
+std::unique_ptr<OI> Robot::oi;
+
 void Robot::RobotInit() {
-	RobotMap::init();
-	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+  RobotMap::init();
+  // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
 
   // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
-	// This MUST be here. If the OI creates Commands (which it very likely
-	// will), constructing it during the construction of CommandBase (from
-	// which commands extend), subsystems are not guaranteed to be
-	// yet. Thus, their requires() statements may grab null pointers. Bad
-	// news. Don't move it.
+  // This MUST be here. If the OI creates Commands (which it very likely
+  // will), constructing it during the construction of CommandBase (from
+  // which commands extend), subsystems are not guaranteed to be
+  // yet. Thus, their requires() statements may grab null pointers. Bad
+  // news. Don't move it.
   Chassis::getInstance();
   OI::getInstance();
```

Comment:
Did you guys read the comments above? Create OI at the very end, after all Subsystems